controllers/managedocs_controller.go

```diff
@@ -920,6 +955,39 @@ func (r *ManagedOCSReconciler) reconcileOnboardingValidationSecret() error {
 	return nil
 }
 
+func (r *ManagedOCSReconciler) reconcileCSIKMSConnectionDetailsConfigMap() error {
+	if r.DeploymentType != consumerDeploymentType {
+		r.Log.Info(fmt.Sprintf("Non consumer deployment, skipping reconcile for %s config map", csiKMSConnectionDetailsConfigMapName))
+		return nil
+	}
+	r.Log.Info(fmt.Sprintf("Reconciling %s configMap", csiKMSConnectionDetailsConfigMapName))
+
+	_, err := ctrl.CreateOrUpdate(r.ctx, r.Client, r.csiKMSConnectionDetailsConfigMap, func() error {
+		if err := r.own(r.csiKMSConnectionDetailsConfigMap); err != nil {
+			return err
+		}
+
+		awsStsMetadataTestAsString := string(
+			testutils.ToJsonOrDie(
```

Comment:
`testutils` should not get imported into non-test code. The opposite is ok

Reply:
toJsonOrDie exists in testutils but not used anywhere except sts code.
I have moved the func from testutils to utils.